app/views/pages/home.html.erb

```diff
@@ -1,2 +1,18 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(https://raw.githubusercontent.com/ahsanazim/slack-landing-page/master/screen_caps/main_background.jpg); height: 100vh">
+  <div class="container">
+    <h1 style= "color:#F7B538;">Satisfy your curiousity and discover your power</h1>
+    <p style= "color:#F7B538;">An incredible variety of abilities available at the click of a button</p>
```

Comment:
change this as well to something like:
"Discover and exchange superpowers to satisfy your current needs!"